python/ansible/plugins/action/adcm_check.py

```diff
@@ -77,18 +77,35 @@ def run(self, tmp=None, task_vars=None):
 
         result = super(ActionModule, self).run(tmp, task_vars)
```

Comment:
this string should be removed, it is no longer nessesary

Reply:
Ok